Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:lts-slim
```

Comment:
Can you pin it to 16 lts slim please? That way the version we get is deterministic.

Reply:
should be fine now